docs/providers/azure/events/servicebus.md

```diff
@@ -33,7 +33,7 @@ functions:
     handler: handler.hello
     events:
       - serviceBus:
-        name: item #<string>, default - "mySbMsg", specifies which name is available on `context.bindings`
+          name: item #<string>, default - "mySbMsg", specifies which name is available on `context.bindings`
```

Comment:
Is that the only property that should be indented? What about the others below?

Reply:
Is the only one, it works as expected with that indentation @pgrzesik